handbook/company/pricing-features-table.yml

```diff
@@ -527,6 +527,9 @@
     - industryName: Labels (SQL-driven)
       friendlyName: Filter hosts using SQL
       tier: Free
+    - industryName: Custom device data for help desk
```

Comment:
Mike: Can you add product categories and department.

Reply:
✅